### DISCUSSION TITLE
Revolt Account

### DISCUSSION THREAD

**Discussion by KarmaP0lice** [stoatchat/discussions#614] [Q&A] [answered]
I've been trying to log in but the verification e-mail never shows up. Is that normal? I've been waiting approximately one hour. Do I have to wait more?

**Comment by Rexogamer** [comment]
A few questions:
- If you're comfortable saying, are you using Outlook or iCloud?
- Have you checked your spam folder?
- Have you tried resending it?

**Reply by KarmaP0lice** [reply]
I'm using Outlook. And yes, I've tried to resend it and I even try to use another email, but still it doesn't work.

**Reply by Rexogamer** [reply, marked as answer]
Ahh, that'd do it - we're currently experiencing some issues with Outlook/iCloud blocking our emails. We're trying to get this resolved, but for now you might want to try another provider?

**Reply by KarmaP0lice** [reply]
Like Google? Let me check it out
